litellm/types/utils.py

```diff
@@ -970,9 +970,9 @@ def json(self, **kwargs):  # type: ignore
 
 class Logprobs(OpenAIObject):
```

Comment:
is there a reason to not use the openai sdk's pydantic logprobs object here? @ishaan-jaff

Reply:
OpenAI defines their Logprobs object for TextCompletion like this - which was the cause of the problem. Their completion API and type is not in sync 

```

class Logprobs(BaseModel):
    text_offset: Optional[List[int]] = None

    token_logprobs: Optional[List[float]] = None

    tokens: Optional[List[str]] = None

    top_logprobs: Optional[List[Dict[str, float]]] = None
```